test-basic/lockForUpdate-test.js

```diff
@@ -98,8 +98,8 @@ describe('optic-update lockForUpdate tests', function() {
         it('test with fromParam with custom uri', function (done) {
             const rows = [{myUri: '/optic/test/musician1.json'}];
             const outputCols = [{"column": "myUri", "type": "string", "nullable": false}];
-
-            db.rows.query(op.fromParam('bindingParam', null, outputCols).lockForUpdate(op.col('myUri')), null, {bindingParam: rows}).then((res) => {
+            const options = serverConfiguration.serverVersion <= 11.1 ? null:{ opticResponseStructure: 'object'};
```

Comment:
I don't think you need to introduce `opticResponseStructure` - the Node Client already supports `structure` as an option, you should be able to use that. 

For example, this test in rows.js is doing just that:

```
it('as a JSON array', function(){
  return db.rows.query(planFromJSON, {queryType: 'json', format: 'json', structure: 'array'})
    .then(function(response) {
      valcheck.isArray(response).should.equal(true);
    });
});
```